qutebrowser/browser/adblock.py

```diff
@@ -270,6 +267,7 @@ def _merge_file(self, byte_io):
             line_count += 1
             ok = self._parse_line(line)
             if not ok:
+                message.error("adblock: read error occurred for line: {}".format(line))
```

Comment:
That should probably only go to the log - otherwise you get spammed with *lots* of messages when loading a file which isn't /etc/hosts-like at all.

Reply:
oh whoops!